app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt

```diff
@@ -44,20 +44,34 @@ object DraftController {
     private const val REGEX_FORWARD = "(fw|fwd|rv|wg|tr|i):"
 
     //region Queries
-    private fun MutableRealm?.getDraftsQuery(): RealmQuery<Draft> {
-        return (this ?: RealmDatabase.mailboxContent()).query()
+    private fun MutableRealm?.getDraftsQuery(query: String? = null): RealmQuery<Draft> {
+        return with(this ?: RealmDatabase.mailboxContent()) {
+            query?.let { query(it) } ?: query()
+        }
```

Comment:
At no time is `MutableRealm` null. So we have to simplify this

Reply:
It is null when get drafts with actions